### ISSUE TITLE
Timer beeping strikes back ;-)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MikeBland/er9x#155]
```
What steps will reproduce the problem?
Set a count down

What is the expected output?
At the end, the timer restarts in the "up way", beeping the same way than 
during "down" phase (10 sec : one beep, 20 : two, etc...

What do you see instead?
One long minute with a beep each second.

What version of the product are you using? On what operating system?
231

Please provide any additional information below.


```

Original issue reported on code.google.com by `CBx...@gmail.com` on 1 Nov 2010 at 12:08

**Comment by GoogleCodeExporter** [comment]
```
Hi,

I couldn't reproduce the problem.  Could you try with a more recent revision?
Can you also describe your setup (or post an eeprom file)?
```

Original comment by `erezra...@gmail.com` on 1 Nov 2010 at 4:10

**Comment by GoogleCodeExporter** [comment]
```
Hello,

Hello,

Here a file. Try the "fitness" model.

The timer is set on the trn switch.

I tried with the revision 244, and the timer beeps still a complete minute 
after the end of the count down.




```

Original comment by `CBx...@gmail.com` on 1 Nov 2010 at 5:36

Attachments:
- [svg-20101101.hex](https://storage.googleapis.com/google-code-attachments/er9x/issue-155/comment-2/svg-20101101.hex)

**Comment by GoogleCodeExporter** [comment]
```
Oh, I got it wrong.
I though you had the timer beeping in reverse when you WANT it to.

This is kind of problematic.  Not hard to do but rather not within my 
conception of a timer.
Let me "sleep on it" :D
```

Original comment by `erezra...@gmail.com` on 1 Nov 2010 at 6:34

**Comment by GoogleCodeExporter** [comment]
```
May be the timer just should not beep continuously during one minute, but beep 
the same way than during the count down (10 sec : one beep, 20 : two, etc...

```

Original comment by `CBx...@gmail.com` on 1 Nov 2010 at 8:59

**Comment by GoogleCodeExporter** [comment]
```
A compromise? 

How about things work exactly as now, but add that the trigger switch (assuming 
it's a switch) could semi-silence the alarm, then leaving the beeps doing 1, 2, 
3, 4, 5 bips for 10,20,30,40,50 seconds? Perhaps extend that to a one long beep 
at one minute, then reverting to 1, 2, 3 bips etc, and two long beeps at 2 
minutes, and so forth?

In this way, we might know how long we have been flying beyond the alarm point, 
without looking at the screen.

In other words, a new 'Alarm recognised. Now counting up' state for the alarm 
could be entered if the trigger switch is activated once in the 'zero-reached 
alarm' state.
```

Original comment by `gru...@gmail.com` on 15 Nov 2010 at 10:01

**Comment by GoogleCodeExporter** [comment]
```
Yes. It seems (and sounds ;-) ) good to proceed like that.

Thx.
```

Original comment by `CBx...@gmail.com` on 16 Nov 2010 at 7:31

**Comment by GoogleCodeExporter** [comment]
Original comment by `erezra...@gmail.com` on 6 Dec 2010 at 6:28
- Changed state: **WontFix**
